04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,25 @@
-#write your code here
+
+require 'pry'
+require 'pry-byebug'
+
+def translate(words)
+    words.split(" ").map {|word| translate_word(word)}.join(" ")
+end
+
+def translate_word(word)
+    first_letter = word[0].downcase
+    
+    if ["a", "e", "i", "o", "u"].include?(first_letter)
```

Comment:
we can have a variable like
`vowels = ["a", "e", "i", "o", "u"]`
and reuse it in the following lines of code